sycl/test-e2e/Reduction/regression_after_pr_6343.cpp

```diff
@@ -1,14 +1,14 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-#include <sycl/sycl.hpp>
+#include <sycl/detail/core.hpp>
+
+#include <sycl/reduction.hpp>
 
 using namespace sycl;
 
 int main() {
-  device d(default_selector_v);
-  context ctx{d};
-  queue q{ctx, d};
+  queue q;
```

Comment:
has this been tested?  I've seen this device..context..queue pattern before and it is used because on PVC with the new FLAT mode, the default selector will select all the L0 GPUs on the platform, which is two.  So the queue gets created with a context that has two devices.  Then, this sometimes leads to problems.  The code you are replacing guarantees a context of one device. 

I believe some of those problems have since been addressed, but not all.  So we need to make sure this test change works on PVC.

Reply:
Yes, I've been developing the patch on PVC. Also, whenever this craziness is done on purpose, it better be properly commented.